spec/app/models/metasploit/cache/license_spec.rb

```diff
@@ -0,0 +1,34 @@
+RSpec.describe Metasploit::Cache::License do
+  context 'database' do
+    context 'columns' do
+      it { is_expected.to have_db_column(:abbreviation).of_type(:string).with_options(null: false, unique: true) }
+      it { is_expected.to have_db_column(:summary).of_type(:text).with_options(null: false, unique: true) }
+      it { is_expected.to have_db_column(:url).of_type(:string).with_options(null: false, unique: true) }
+    end
```

Comment:
You should add a `"indices"` `context` and use `have_db_index` to check for the indexes you'll create in https://github.com/rapid7/metasploit-cache/pull/43/files#r30547368.

```ruby
RSpec.describe Metasploit::Cache::License do
  context 'database' do
    context 'columns' do
      # ...
    end

    context 'indices' do
      it { is_expected.to have_db_index(:abbreviation).unique(true) }
      it { is_expected.to have_db_index(:summary).unique(true) }
      it { is_expected.to have_db_index(:url).unique(true) }
    end
  end
  # ...
end
```